env-example

```diff
@@ -1,6 +1,6 @@
-DOVETAIL_HOST=
-DOVETAIL_PROD_HOST=
-FEEDER_HOST=
-PUBLISH_HOST=
+DOVETAIL_HOST=dovetail-staging.prxu.org
+DOVETAIL_PROD_HOST=dovetail.prxu.org
+FEEDER_HOST=feeder-staging.prx.tech
+PUBLISH_HOST=publish-staging.prx.tech
 PUBLISH_USER=
```

Comment:
should we set a default publish_user and publish_password that will have access, to make tests simpler to run?